src/sentry/web/frontend/auth_login.py

```diff
@@ -109,25 +531,14 @@ def get_join_request_link(self, organization):
 
         return reverse("sentry-join-request", args=[organization.slug])
 
-    def get_next_uri(self, request: Request):
-        next_uri_fallback = None
-        if request.session.get("_next") is not None:
-            next_uri_fallback = request.session.pop("_next")
-        return request.GET.get(REDIRECT_FIELD_NAME, next_uri_fallback)
-
-    def get_post_register_url(self, request: Request):
-        base_url = auth.get_login_redirect(request)
-        params = {"frontend_events": json.dumps({"event_name": "Sign Up"})}
-        return add_params_to_url(base_url, params)
-
-    def respond_login(self, request: Request, context, **kwargs):
-        return self.respond("sentry/login.html", context)
-
-    def _handle_login(self, request: Request, user, organization: Optional[Organization]):
-        login(request, user, organization_id=coerce_id_from(organization))
-        self.determine_active_organization(request)
-
-    def handle_basic_auth(self, request: Request, **kwargs):
+    def handle_basic_auth(
```

Comment:
Are you still keeping this method around? It doesn't seem to be used anymore.

Reply:
It's used in `auth_organization_login.py`, in the next PR I'll figure out what specifically it's doing and send it to the right function so we can get rid of `handle_basic_auth` soon. 

Since it's such a generalized function, `auth_organization_login` could basically just toss requests at it and the function would figure out what to do, now that functions are more specific, calls will need to be routed properly, which I'll do right after this